handbook/company/communications.md

```diff
@@ -191,7 +191,10 @@ When a contract is going to be routed for signature by someone outside of Fleet
 
 The SLA for contract signature is **2 business days**. Please do not follow up on signature unless this time has elapsed.
 
-> _**Note:** Signature open time for the CEO is not currently measured, to avoid the overhead of creating separate signature issues to measure open and close time. This may change as signature volume increases._
+_**Note:** Signature open time for the CEO is not currently measured, to avoid the overhead of creating separate signature issues to measure open and close time. This may change as signature volume increases._
+
+> _**Note:** If a contract is ready for signature and requires no review or revision, the requestor logins into DocuSign using hello@ from the 1Password and routes the agreement to the CEO for signature. 
+Please use [Fleet's billing email address](https://fleetdm.com/handbook/company/communications#email-relays) for all contracts, and never use individual emails except for signature.  If the page to sign includes any individual emails in the docusign contract, please remove it before routing to the CEO for signature.
```

Comment:
1. From the markdown alone, it's unclear to me here whether line 197 is supposed to be inside of the note or not.

2. Also looks like the italics begun on line 196 doesn't close